exampleCourse/questions/demo/annotated/engLogic/question.html

```diff
@@ -5,7 +5,7 @@
 <br>
 </pl-question-panel>
 
-<p>
+<div class="mb-3">
```

Comment:
`pl-overlay` doesn't have a bottom margin.

Reply:
Nit: move this to inside `pl-hide-in-panel`, so that the answer panel doesn't have an empty div?